replications/internal/queue_management.go

```diff
@@ -116,13 +122,6 @@ func (rq *replicationQueue) Close() error {
 	return rq.queue.Close()
 }
 
-// WriteFunc is currently a placeholder for the "default" behavior
-// of the queue scanner sending data from the durable queue to a remote host.
-func WriteFunc(b []byte) error {
-	// TODO: Add metrics updates for BytesSent, BytesDropped, and ErrorCodes
```

Comment:
Metrics still do need to be finished off, issue tracking this is: https://github.com/influxdata/influxdb/issues/22734